polygeist/tools/cgeist/Lib/ValueCategory.h

```diff
@@ -99,6 +102,7 @@ class ValueCategory {
                      mlir::Value RHS, bool IsExact = false) const;
   ValueCategory ExactSDiv(mlir::OpBuilder &Builder, mlir::Location Loc,
                           mlir::Value RHS) const;
+  ValueCategory FNeg(mlir::OpBuilder &Builder, mlir::Location Loc) const;
```

Comment:
I think we should pass the value to be negated in the parameter list.

Reply:
See above